catalog/content/core/content-core-catalogerplugin/pom.xml

```diff
@@ -81,14 +90,13 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>guava</Embed-Dependency>
+                        <Embed-Dependency>ddf-security-common,platform-util-unavailableurls</Embed-Dependency>
                         <Private-Package>
                             ddf.content.resource.impl,
                             ddf.catalog.data.impl,
```

Comment:
It seems safe but I don't really want to make that change since I'm not actually working on the code in that area and don't want to have to figure out how to test it properly.